src/main/scala/org/broadinstitute/dsde/firecloud/utils/RestJsonClient.scala

```diff
@@ -49,33 +51,49 @@ trait RestJsonClient extends FireCloudRequestBuilding {
     }
 
     val finalPipeline = addCreds.map(creds => creds ~> pipeline).getOrElse(pipeline)
-    finalPipeline(req)
+    finalPipeline(req) map { res =>
+      if (label.nonEmpty && tick != -1) {
```

Comment:
Given the definition of `tick`, couldn't the predicate be `tick != -1` only?

Reply:
yup, it can; done